<ui version="4.0" >
 <class>ServerWindow</class>
 <widget class="QMainWindow" name="ServerWindow" >
  <property name="enabled" >
   <bool>true</bool>
  </property>
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>660</width>
    <height>578</height>
   </rect>
  </property>
  <property name="sizePolicy" >
   <sizepolicy>
    <hsizetype>5</hsizetype>
    <vsizetype>5</vsizetype>
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle" >
   <string>AdderGUI</string>
  </property>
  <property name="windowIcon" >
   <iconset resource="../resource.qrc" >:/images/addericon64x64.png</iconset>
  </property>
  <property name="toolButtonStyle" >
   <enum>Qt::ToolButtonTextOnly</enum>
  </property>
  <widget class="QWidget" name="centralwidget" >
   <layout class="QGridLayout" >
    <property name="margin" >
     <number>9</number>
    </property>
    <property name="spacing" >
     <number>6</number>
    </property>
    <item row="1" column="0" >
     <widget class="QTreeWidget" name="procedureTreeWidget" >
      <property name="cursor" >
       <cursor>0</cursor>
      </property>
      <property name="whatsThis" >
       <string>&lt;html>&lt;head>&lt;meta name="qrichtext" content="1" />&lt;/head>&lt;body style=" white-space: pre-wrap; font-family:Sans Serif; font-size:9pt; font-weight:400; font-style:normal; text-decoration:none;">&lt;p style="-qt-paragraph-type:empty; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;">&lt;/p>&lt;/body>&lt;/html></string>
      </property>
      <property name="autoFillBackground" >
       <bool>true</bool>
      </property>
      <property name="frameShape" >
       <enum>QFrame::StyledPanel</enum>
      </property>
      <property name="frameShadow" >
       <enum>QFrame::Sunken</enum>
      </property>
      <property name="alternatingRowColors" >
       <bool>true</bool>
      </property>
      <property name="rootIsDecorated" >
       <bool>false</bool>
      </property>
      <property name="sortingEnabled" >
       <bool>false</bool>
      </property>
      <property name="columnCount" >
       <number>2</number>
      </property>
      <column>
       <property name="text" >
        <string>0</string>
       </property>
      </column>
      <column>
       <property name="text" >
        <string>1</string>
       </property>
      </column>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar" >
   <property name="geometry" >
    <rect>
     <x>0</x>
     <y>0</y>
     <width>660</width>
     <height>31</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuHelp" >
    <property name="title" >
     <string>&amp;Help</string>
    </property>
    <addaction name="action_About" />
    <addaction name="actionAbout_Qt" />
   </widget>
   <widget class="QMenu" name="menuServer" >
    <property name="title" >
     <string>&amp;Server</string>
    </property>
    <addaction name="action_Connect" />
    <addaction name="action_Disconnect" />
    <addaction name="separator" />
    <addaction name="action_Refresh" />
    <addaction name="separator" />
    <addaction name="action_Quit" />
   </widget>
   <widget class="QMenu" name="menu_Settings" >
    <property name="title" >
     <string>&amp;Tools</string>
    </property>
    <addaction name="action_Create_Users" />
    <addaction name="action_Open_Console" />
    <addaction name="separator" />
    <addaction name="action_Options" />
   </widget>
   <widget class="QMenu" name="menu_Procedure" >
    <property name="enabled" >
     <bool>true</bool>
    </property>
    <property name="title" >
     <string>&amp;Procedure</string>
    </property>
    <widget class="QMenu" name="menuAdministrative_Actions" >
     <property name="title" >
      <string>&amp;Administrative</string>
     </property>
     <addaction name="action_Start_Procedure" />
     <addaction name="action_Reset_Procedure" />
     <addaction name="action_Delete_Procedure" />
     <addaction name="separator" />
     <addaction name="action_Create_Procedure" />
    </widget>
    <addaction name="actionViewStatus" />
    <addaction name="action_Participate" />
    <addaction name="separator" />
    <addaction name="menuAdministrative_Actions" />
   </widget>
   <addaction name="menuServer" />
   <addaction name="menu_Procedure" />
   <addaction name="menu_Settings" />
   <addaction name="menuHelp" />
  </widget>
  <widget class="QStatusBar" name="statusbar" />
  <widget class="QToolBar" name="toolBar" >
   <property name="orientation" >
    <enum>Qt::Horizontal</enum>
   </property>
   <property name="toolButtonStyle" >
    <enum>Qt::ToolButtonIconOnly</enum>
   </property>
   <attribute name="toolBarArea" >
    <number>4</number>
   </attribute>
   <addaction name="action_Connect" />
   <addaction name="action_Disconnect" />
   <addaction name="separator" />
   <addaction name="actionViewStatus" />
   <addaction name="action_Participate" />
   <addaction name="separator" />
   <addaction name="action_Create_Procedure" />
   <addaction name="action_Start_Procedure" />
   <addaction name="action_Reset_Procedure" />
   <addaction name="action_Delete_Procedure" />
   <addaction name="separator" />
  </widget>
  <action name="action_Connect" >
   <property name="icon" >
    <iconset resource="../resource.qrc" >:/images/connect_creating.png</iconset>
   </property>
   <property name="text" >
    <string>&amp;Connect...</string>
   </property>
   <property name="statusTip" >
    <string>Connect to a server.</string>
   </property>
  </action>
  <action name="action_Disconnect" >
   <property name="enabled" >
    <bool>false</bool>
   </property>
   <property name="icon" >
    <iconset resource="../resource.qrc" >:/images/connect_no.png</iconset>
   </property>
   <property name="text" >
    <string>&amp;Disconnect</string>
   </property>
   <property name="statusTip" >
    <string>Disconnect from the server.</string>
   </property>
  </action>
  <action name="action_Quit" >
   <property name="text" >
    <string>&amp;Quit</string>
   </property>
   <property name="statusTip" >
    <string>Exit the program.</string>
   </property>
   <property name="shortcut" >
    <string>Ctrl+Q</string>
   </property>
  </action>
  <action name="action_About" >
   <property name="icon" >
    <iconset resource="../resource.qrc" >:/images/addericon64x64.png</iconset>
   </property>
   <property name="text" >
    <string>&amp;About AdderGUI</string>
   </property>
   <property name="statusTip" >
    <string>Display ADDER Client copyright information.</string>
   </property>
  </action>
  <action name="actionAbout_Qt" >
   <property name="text" >
    <string>About Qt</string>
   </property>
   <property name="statusTip" >
    <string>Display Qt copyright information.</string>
   </property>
  </action>
  <action name="action_Create_Procedure" >
   <property name="enabled" >
    <bool>false</bool>
   </property>
   <property name="icon" >
    <iconset resource="../resource.qrc" >:/images/wizard.png</iconset>
   </property>
   <property name="text" >
    <string>&amp;Create Procedure...</string>
   </property>
   <property name="statusTip" >
    <string>Create a new procedure.</string>
   </property>
  </action>
  <action name="action_Delete_Procedure" >
   <property name="enabled" >
    <bool>false</bool>
   </property>
   <property name="icon" >
    <iconset resource="../resource.qrc" >:/images/editdelete.png</iconset>
   </property>
   <property name="text" >
    <string>&amp;Delete</string>
   </property>
   <property name="statusTip" >
    <string>Delete the selected procedure.</string>
   </property>
  </action>
  <action name="action_Advance_Procedure" >
   <property name="enabled" >
    <bool>false</bool>
   </property>
   <property name="icon" >
    <iconset resource="../resource.qrc" >:/images/player_end.png</iconset>
   </property>
   <property name="text" >
    <string>&amp;Advance</string>
   </property>
   <property name="statusTip" >
    <string>Advance the stage of the selected procedure.</string>
   </property>
  </action>
  <action name="action_Reset_Procedure" >
   <property name="enabled" >
    <bool>false</bool>
   </property>
   <property name="icon" >
    <iconset resource="../resource.qrc" >:/images/start.png</iconset>
   </property>
   <property name="text" >
    <string>&amp;Reset</string>
   </property>
   <property name="statusTip" >
    <string>Reset the selected procedure.</string>
   </property>
  </action>
  <action name="action_Start_Procedure" >
   <property name="enabled" >
    <bool>false</bool>
   </property>
   <property name="icon" >
    <iconset resource="../resource.qrc" >:/images/player_play.png</iconset>
   </property>
   <property name="text" >
    <string>&amp;Advance/Start</string>
   </property>
   <property name="statusTip" >
    <string>Advance the selected procedure.</string>
   </property>
  </action>
  <action name="actionViewStatus" >
   <property name="enabled" >
    <bool>false</bool>
   </property>
   <property name="icon" >
    <iconset resource="../resource.qrc" >:/images/viewmag.png</iconset>
   </property>
   <property name="text" >
    <string>&amp;View Status...</string>
   </property>
   <property name="statusTip" >
    <string>View the selected procedure's information.</string>
   </property>
  </action>
  <action name="action_Participate" >
   <property name="enabled" >
    <bool>false</bool>
   </property>
   <property name="icon" >
    <iconset resource="../resource.qrc" >:/images/kontact_contacts.png</iconset>
   </property>
   <property name="text" >
    <string>&amp;Participate...</string>
   </property>
  </action>
  <action name="action_Refresh" >
   <property name="enabled" >
    <bool>false</bool>
   </property>
   <property name="text" >
    <string>&amp;Refresh</string>
   </property>
   <property name="shortcut" >
    <string>F5</string>
   </property>
  </action>
  <action name="actionViewVoters" >
   <property name="enabled" >
    <bool>false</bool>
   </property>
   <property name="text" >
    <string>&amp;Voters...</string>
   </property>
  </action>
  <action name="actionViewAuthorities" >
   <property name="enabled" >
    <bool>false</bool>
   </property>
   <property name="text" >
    <string>&amp;Authorities...</string>
   </property>
  </action>
  <action name="action_Open_Console" >
   <property name="enabled" >
    <bool>true</bool>
   </property>
   <property name="text" >
    <string>Co&amp;nsole...</string>
   </property>
  </action>
  <action name="action_Options" >
   <property name="text" >
    <string>&amp;Options...</string>
   </property>
  </action>
  <action name="action_Create_Users" >
   <property name="enabled" >
    <bool>false</bool>
   </property>
   <property name="text" >
    <string>&amp;Create Users...</string>
   </property>
  </action>
 </widget>
 <tabstops>
  <tabstop>procedureTreeWidget</tabstop>
 </tabstops>
 <resources>
  <include location="../resource.qrc" />
 </resources>
 <connections>
  <connection>
   <sender>action_Quit</sender>
   <signal>triggered()</signal>
   <receiver>ServerWindow</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel" >
     <x>286</x>
     <y>249</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
